api/URL.json

```diff
@@ -982,10 +982,10 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/URL/toString",
           "support": {
             "chrome": {
-              "version_added": "52"
+              "version_added": "37"
             },
             "chrome_android": {
-              "version_added": "52"
+              "version_added": "37"
```

Comment:
```suggestion
              "version_added": "25"
```

Reply:
Interesting -- so this feature arrived a few versions later for the `webkitURL` in Chrome for Android than the `webkitURL` in desktop Chrome? (You didn't mention this version in your overall comment.)

Reply:
Actually, it’s because Chrome Android’s first release was version 18, and then its second was 25, so there were no in-betweens.

Reply:
Ahh, that makes sense!